### PR TITLE
Added descriptions for the parameters and return value

### PR DESCRIPTION
**Pull request by Umbrella-Hanseo-Jo**
Summary of the modifications:

1. <param name="source">: Added a description for "source".
2. <param name="keySelector">: Added a description for "keySelector".
3. <typeparam name="TSource">: Added a description for "TSource".
4. <typeparam name="TKey">: Added a description for "TKey".
5. <returns>: Added a description for the return value.

Expected benefits:

With these modifications, code users will have a clearer understanding of the parameters and return value of this function.

### CODE REVIEW

**Comment by SoniaComp** [comment]
LGTM 😊

**Review by justinyoo — CHANGES_REQUESTED** [review]
Thanks for your PR! Could you please update your XML comments like what the other XML comments do, for the consistency purpose?

File: src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs [hunk 1]
```diff
@@ -14,11 +14,11 @@ public static class EnumerableExtensions
         /// <summary>
         /// Distinct by defined property
         /// </summary>
-        /// <param name="source"></param>
-        /// <param name="keySelector"></param>
-        /// <typeparam name="TSource"></typeparam>
-        /// <typeparam name="TKey"></typeparam>
-        /// <returns></returns>
+        /// <param name="source"><see cref="IEnumerable"/> instance.</param>
```
Comment (justinyoo) [inline src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs:17]: `IEnumerable{TSource}`

File: src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs [hunk 2]
```diff
@@ -14,11 +14,11 @@ public static class EnumerableExtensions
         /// <summary>
         /// Distinct by defined property
         /// </summary>
-        /// <param name="source"></param>
-        /// <param name="keySelector"></param>
-        /// <typeparam name="TSource"></typeparam>
-        /// <typeparam name="TKey"></typeparam>
-        /// <returns></returns>
+        /// <param name="source"><see cref="IEnumerable"/> instance.</param>
+        /// <param name="keySelector">Func<TSource, TKey>) instance.</param>
```
Comment (justinyoo) [inline src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs:18]: `<see cref="Func{TSource, TKey}"/> instance.`

File: src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs [hunk 3]
```diff
@@ -14,11 +14,11 @@ public static class EnumerableExtensions
...
         /// Distinct by defined property
         /// </summary>
-        /// <param name="source"></param>
-        /// <param name="keySelector"></param>
-        /// <typeparam name="TSource"></typeparam>
-        /// <typeparam name="TKey"></typeparam>
-        /// <returns></returns>
+        /// <param name="source"><see cref="IEnumerable"/> instance.</param>
+        /// <param name="keySelector">Func<TSource, TKey>) instance.</param>
+        /// <typeparam name="TSource"><see cref="IEnumerable{TSource}"/> instance.</typeparam>
```
Comment (justinyoo) [inline src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs:19]: `Type of <see cref="IEnumerable{T}" />.`

File: src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs [hunk 4]
```diff
@@ -14,11 +14,11 @@ public static class EnumerableExtensions
...
+        /// <typeparam name="TKey"><see cref="Func{TSource, TKey}"/> instance.</typeparam>
```
Comment (justinyoo) [inline src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs:20]: `Type of <see cref="Func{T1, T2}" />.`

File: src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs [hunk 5]
```diff
@@ -14,11 +14,11 @@ public static class EnumerableExtensions
...
+        /// <returns>Returns an <see cref="IEnumerable{TSource}"/> that contains the distinct elements based on the specified property.</returns>
```
Comment (justinyoo) [inline src/Microsoft.Azure.Functions.Worker.Extensions.OpenApi/Extensions/EnumerableExtensions.cs:21]: Returns the...

**Review by justinyoo — CHANGES_REQUESTED** [review]

**Review by justinyoo — APPROVED** [review]
LGTM! Thanks for your contribution

**Pull request merged**